docs/sources/dashboards/create-reports/_index.md

```diff
@@ -0,0 +1,331 @@
+   - [Format report](#2-format-report)
+   - [Schedule](#3-schedule)
+   - [Share](#4-share)
+   - [Confirm](#5-confirm)
+1. Click one of the following buttons in the top-right corner of the screen:
+   - **Send** or **Schedule send** - The report is sent according the schedule you've set.
+   - **Save as draft** - You can save a draft at any point during the report creation or update process, even if it's missing required fields. The report won't be sent according to its schedule while it's a draft.
+   - **Discard** - Delete the report draft. This action can't be reversed.
+
+### 1. Select dashboard
+
+At this step, select the dashboard or dashboards on which the report is based, as well as the variables and time range for those dashboards. The options are:
+
+<!-- prettier-ignore-start -->
+
+| Option | Description |
+| ------ | ----------- |
+| Source dashboard | Select the dashboard from which you want to generate the report. |
+| [Template variables](#template-variables) | Select the variable **Datasource** and **Instance** for the selected dashboard. This option is only displayed if the dashboard has variables. |
+| [Time range (optional)](#time-range) | By default, reports use the saved time range of the dashboard. Optionally, you can change the time range of the report. |
+| Add another dashboard | Add more dashboards to the report. Additional dashboards will be rendered as new pages in the same PDF file, or additional images if you chose to embed images in your report email. You can't add the same dashboard to a report multiple times. |
+
+<!-- prettier-ignore-end -->
+
+#### Template variables
+
+Select the variable **Datasource** and **Instance** for the selected dashboard. This option is only displayed if the dashboard has variables.
+
+You can configure report-specific template variables for the dashboard on the report page. The variables that you select will override the variables from the dashboard, and they are used when rendering a PDF file of the report. For detailed information about using template variables, refer to the [Templates and variables](ref:templates-and-variables) section.
+
+The query variables saved with a report might become out of date if the results of that query change. For example, if your template variable queries for a list of hostnames and a new hostname is added, then it will not be included in the report. If that occurs, the selected variables must be manually updated in the report. If you select the **All** value for the template variable or if you keep the dashboard's original variable selection, then the report stays up-to-date as new values are added.
+
+##### Reports with panels or rows set to repeat by a variable
+
+You can include dynamic dashboards with panels or rows, set to repeat by a variable, into reports. For detailed information about setting up repeating panels or rows in dashboards, refer to [Repeat panels or rows](ref:repeat-panels-or-rows).
+
+#### Time range
```

Comment:
In this section, we should mention how to set a report always using the dashboard time range (the X I mentioned above).